varcode/vcf.py

```diff
@@ -296,6 +301,9 @@ def dataframes_to_variant_collection(
 
     variant_collection_kwargs : dict, optional
         Additional keyword parameters to pass to VariantCollection.__init__.
+
+    sources : set or list
```

Comment:
I think I'm going to leave it out, since the metadata dictionary has gotten somewhat complicated and I'd like to localize that complexity to VariantCollection